io.catenax.specialCharacteristics.measurement/1.0.0/Measurement.ttl

```diff
@@ -0,0 +1,63 @@
+
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.specialCharacteristics.measurement:1.0.0#> .
+@prefix bpn: <urn:samm:io.catenax.shared.business_partner_number:2.0.0#> .
+@prefix samm-u: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix serialPart: <urn:samm:io.catenax.serial_part:3.0.0#> .
+
+:Measurement a samm:Aspect ;
+   samm:preferredName "SpecialCharacteristicMeasurment"@en ;
+   samm:description "Contains the Measurment for a special characteristic"@en ;
+   samm:properties ( serialPart:localIdentifiers bpn:bpnlProperty :customerPartId :characteristicId :results [ samm:property :measurementType; samm:optional true ] :revisionIndex ) ;
```

Comment:
Is the **bpnlProperty** really needed? The same information is stored in the _manufacturerID_ of the _LocalIdentifier_? This information would therefore be redundant.

Unless you don't mean the manufactuererId with the bpnlProperty? If that is the case, then I would still suggest using a unique name for this property to know exactly what is meant.

Reply:
> good point, will discuss this with Jochen and Sebastian. If we move the attribute into local identifier then we should explicitly reference them in the KIT

same here

Reply:
Yes the bpnlProerty must be the manufacturerID. We put it here because it is necessary to identify the related specification of the characteristic. Yes, the local identfier **may** contain the manufacturerId but it is not enforced by the datamodel. If we should use the localIdentifier for the manufacturerId this should be mentioned in the KIT and noted that this information is neccessary